doc/Extensions/Dispatcher-Service.md

```diff
@@ -82,9 +82,13 @@ Once you have your Redis database set up, configure it in the .env file on each
 
 ```dotenv
 REDIS_HOST=127.0.0.1
-#REDIS_DB=0
+REDIS_PORT=6379
```

Comment:
Probably best to split this into two sections instead of two in one seciton.

Reply:
As in, split the docs into two different indent blocks, or split via spacing inside the single indent block?